### PR TITLE
Feat(ui/dataLoaders): Change plugin selection to bundle selection

### LINKED ISSUE #1655: dataLoaders: replace list of streaming sources with list of streaming bundles
as a new user
with admin permissions or higher
when i view the list of streaming sources
then i would like to view the config bundles, not the sources.

the first one is:
- System (contains these plugins: cpu, disk, diskio, kernel, mem, processes, swap, system)
- more tbd

### PR DESCRIPTION
**Pull request by ischolten**
Closes #1655

_Briefly describe your proposed changes:_
Instead of individually adding plugins, users will select a bundle of plugins.

  - [x] Rebased/mergeable
  - [x] Tests pass

### CODE REVIEW

File: ui/mocks/dummyData.ts [hunk 1]
```diff
@@ -289,7 +299,41 @@ export const cpuPlugin = {
 export const telegrafPlugin = {
   name: TelegrafPluginInputCpu.NameEnum.Cpu,
   configured: ConfigurationState.Unconfigured,
-  active: true,
+  active: false,
+}
+
+export const cpuTelegrafPlugin = {...telegrafPlugin}
```
Comment (AlirieGray) [inline ui/mocks/dummyData.ts]: nit: what do you think about spacing out these constants?

File: ui/mocks/dummyData.ts [hunk 2]
```diff
@@ -348,6 +392,10 @@ export const telegrafConfigsResponse = {
   request: {},
 }
 
+export const createTelegrafConfigResponse = {
+  data: telegrafConfig,
```
Comment (AlirieGray) [inline ui/mocks/dummyData.ts]: just to see if I'm understanding correctly, this is referring to `telegrafConfig` above which has name `TelegrafPluginInputCpu.NameEnum.Cpu`? is there a reason?

Comment (ischolten) [reply ui/mocks/dummyData.ts]: correct; its just dummy data. i just want the output to be the the config that i created

File: ui/mocks/dummyData.ts (hunk 1 above)
Comment (ischolten) [reply ui/mocks/dummyData.ts]: can do

File: ui/src/onboarding/utils/pluginConfigs.ts [hunk 3]
```diff
@@ -27,3 +31,30 @@ export const updateConfigFields = <T extends Plugin>(
 export const createNewPlugin = (name: TelegrafPluginName): Plugin => {
   return telegrafPluginsInfo[name].defaults
 }
+
+export const isPluginUniqueToBundle = (
```
Comment (AlirieGray) [inline ui/src/onboarding/utils/pluginConfigs.ts]: nice

File: ui/mocks/dummyData.ts (hunk 2 above)
Comment (AlirieGray) [reply ui/mocks/dummyData.ts]: oh I see, I should have looked at the file name. now I feel silly :P

Comment (ischolten) [reply ui/mocks/dummyData.ts]: all good :)

**Pull request merged**